content/contributor/comms/adapters/simulator.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Simulator Adapter"
+sidebartitle: "Simulator"
+url: "/contributor/comms/adapters/simulator"
+weight: 4
+---
+
+The Simulator adapter is a completely arbitrary implementation, with no specified behavior other than the adapter interface. 
```

Comment:
Is this part of the protocol? I would have though that this was the client's internal implementation

Reply:
This is an excellent point. I added this here because it was in the ADR, but I'm also wondering whether it should be specified or not. It's more of a debugging strategy for implementers, isn't it?

If we wanted to actually support custom behavior, we could replace `simulator:` with a `custom:` transport. If not, I think we should remove this, developers will think of mocking the interface without the need for us adding new URIs.

What do you guys think?